src/components/app.tsx

```diff
@@ -1,414 +1,42 @@
-              onClose={handleClose}
-            >
-              <OmniPanelView id={OmniPanelViewIndex.MainMenu}>
-                <MainMenu
-                  onDoorsClick={handleMainMenuDoorsClick}
-                  onLiftsClick={handleMainMenuLiftsClick}
-                  onRobotsClick={handleMainMenuRobotsClick}
-                  onPlacesClick={handleMainMenuPlacesClick}
-                  onDispensersClick={handleMainMenuDispensersClick}
-                />
-              </OmniPanelView>
-              <OmniPanelView id={OmniPanelViewIndex.Doors}>
-                <DoorsPanel
-                  transport={transport}
-                  doorStates={doorStates}
-                  doors={doors}
-                  spotlight={doorSpotlight}
-                />
-              </OmniPanelView>
-              <OmniPanelView id={OmniPanelViewIndex.Lifts}>
-                <LiftsPanel
-                  transport={transport}
-                  liftStates={liftStates}
-                  lifts={lifts}
-                  spotlight={liftSpotlight}
-                />
-              </OmniPanelView>
-              <OmniPanelView id={OmniPanelViewIndex.Robots}>
-                <RobotsPanel transport={transport} fleets={fleets} spotlight={robotSpotlight} />
-              </OmniPanelView>
-              <OmniPanelView id={OmniPanelViewIndex.Places}>
-                {buildingMap && (
-                  <PlacesPanel buildingMap={buildingMap} spotlight={placeSpotlight} />
-                )}
-              </OmniPanelView>
-              <OmniPanelView id={OmniPanelViewIndex.Dispensers}>
-                <DispensersPanel dispenserStates={dispenserStates} spotlight={dispenserSpotlight} />
-              </OmniPanelView>
-            </OmniPanel>
-          </Fade>
-          <SettingsDrawer
-            settings={settings}
-            open={showSettings}
-            onSettingsChange={newSettings => {
-              setSettings(newSettings);
-              saveSettings(newSettings);
-            }}
-            onClose={() => setShowSettings(false)}
+    <UserContext.Provider value={user}>
+      <BrowserRouter>
+        <Switch>
+          <Route exact={true} path={LOGIN_ROUTE} component={Login} />
+          <Route exact={true} path={REGISTER_ROUTE} component={RegisterForm} />
+          <PrivateRoute
+            redirectToLogin={true}
+            isAuthenticated={!isEmpty(user)}
+            exact={true}
+            component={Dashboard}
+            path={DASHBOARD_ROUTE}
+          />
+          <PrivateRoute
+            redirectToLogin={true}
+            isAuthenticated={!isEmpty(user)}
+            exact={true}
+            path={DEFAULT_ROUTE}
+            component={Dashboard}
           />
-        </div>
-      </SettingsContext.Provider>
-    </React.Fragment>
+          {/* TODO: When the authentication logic is implemented this should be the default route  */}
+          <Route path="*" component={NotFoundPage} />
```

Comment:
Redirecting to the not found page did not work for me in the dev server.

Reply:
Yeah, it will not work for you because the dashboard is set as the default route default.
```
<PrivateRoute
            redirectToLogin={true}
            isAuthenticated={!isEmpty(user)}
            exact={true}
            path={DEFAULT_ROUTE}
            component={Dashboard}
          />
```

I did that to do the routing as invisible as possible until the login backend it's ready